drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -361,16 +361,93 @@ static int rf2xx_set_channel(struct device *dev, u16_t channel)
 	return 0;
 }
 
+/**
```

Comment:
add a commit message detailing a bit what was added (though the title is indeed quite clear, but you need a message anyway)